chapters/operatorsandexpressions.tex

```diff
@@ -355,10 +356,19 @@ \subsection{If-Expressions}\label{if-expressions}
 \subsection{Member Access Operator}\label{member-access-operator}
 
 It is possible to access members of a class instance using dot notation, i.e., the \lstinline!.! operator.
+It is also possible to select a record member of a general expression by enclosing it in parentheses.
+Note that while the selection is applied to an \lstinline[language=grammar]!output-expression-list! in the grammar, it is only semantically valid when the \lstinline[language=grammar]!output-expression-list! represents an expression.
+
+In case the first operand is an array it is seen as a slicing operation, see \cref{slice-operation}.
 
 \begin{example}
-\lstinline!R1.R! for accessing the resistance component \lstinline!R! of resistor \lstinline!R1!.
-Another use of dot notation: local classes which are members of a class can of course also be accessed using dot notation on the name of the class, not on instances of the class.
+The component reference \lstinline!R1.R! accesses the resistance component \lstinline!R! of resistor \lstinline!R1!.
```

Comment:
Lower camel case of instance names would further help separating this from the form in next paragraph, but I don't know what to do with the fact that `R` is such a well established name for the resistance parameter…

Reply:
We will have to improve that later. I guess we should instance rename the component `R1`